portindex2json/portindex2json.tcl

```diff
@@ -2,7 +2,7 @@
 # a list of JSON objects.
 # Written by Joshua Root <jmr@macports.org>, 2017
 # Requires: tclsh with the tcllib 'json' package.
-# Usage: tclsh portindex2json.tcl path/to/PortIndex
+# Usage: tclsh portindex2json.tcl path/to/PortIndex -info key=value
```

Comment:
Usually you need to use `[]` pair for optional parameters. Not ideal either, but maybe something along the line of:
```
tclsh portindex2json.tcl [-info key=value] path/to/PortIndex
```

Reply:
E.g. `man git` has `[-c <name>=<value>]`.

Reply:
> E.g. `man git` has `[-c <name>=<value>]`.

Thank you